File: particle-connect/android/src/main/kotlin/network/particle/flutter/bridge/model/RpcUrl.java
package network.particle.flutter.bridge.model;

import androidx.annotation.Keep;

import com.google.gson.annotations.SerializedName;
@Keep
public class RpcUrl {

    @SerializedName("evm_url")
    public String evmUrl;

    @SerializedName("sol_url")
    public String solUrl;
}

File: particle-base/android/src/main/kotlin/network/particle/base_flutter/model/RpcUrl.kt
package network.particle.base_flutter.model


import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName
@Keep
class RpcUrl {
    @SerializedName("evm_url")
    var evmUrl: String? = null

    @SerializedName("sol_url")
    var solUrl: String? = null
}